Client/SafeBrowsing/GSafeBrowsing.swift

```diff
@@ -0,0 +1,316 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import CommonCrypto
+import Shared
+
+private let log = Logger.browserLogger
+
+class SafeBrowsingClient {
+    private static let apiKey = "DUMMY_KEY"
+    private static let maxBandwidth = 2048
+    private static let maxDiskSpace = 4096
```

Comment:
According to https://developers.google.com/safe-browsing/v4/update-constraints, this would correspond to 16kB. This is far too low to be providing any kind of protection.

How about setting it to 2^20 (i.e. 1048576) so that it uses a maximum of 4MB per threat type?

Reply:
Hmm it doesn't only download 4-byte prefixes though. `4096 * 4` I guess is what you calculated for 16kb?

I can increase the size but any user using the browser would be downloading 4MB each time it updates?

Reply:
It would be 4MB for a full update, but much less for a partial update, which will be the common case for people who use Brave a lot. This is mostly about how much of the device storage we're willing to dedicate to Safe Browsing. From a security perspective, the more the better, but ultimately it's up to Product to make the call.

Reply:
These variables will be renamed. It's not the maximum amount of disk space in bytes that the database will take. It's the maximum amount of Entries the database can hold. For maxBandwith, it's the max amount of results the that can be processed when the call is made.

If I set any of these to `1048576` the app will easily slow down to a crawl.